pkg/nsx/services/vpc/vpc.go

```diff
@@ -173,8 +180,10 @@ func InitializeVPC(service common.Service) (*VPCService, error) {
 	VPCService.VPCNSNetworkConfigStore = VPCNsNetworkConfigStore{
 		VPCNSNetworkConfigMap: make(map[string]string),
 	}
-	//initialize vpc store and ip blocks store
+	//initialize vpc store, lbs store and ip blocks store
 	go VPCService.InitializeResourceStore(&wg, fatalErrors, common.ResourceTypeVpc, nil, VPCService.VpcStore)
+	wg.Add(1)
```

Comment:
could you update line 158 and 161, not here?

Reply:
No. We might need to add a switch here, thus I use a separate wg.Add(1)

Reply:
Do we still need wg here? Or maybe add a comment to explain why we need a seperate wg?

Reply:
This separate `wg.Add(1)` is easy to refactor, let's keep existing implementation and refactor in future if needed.